rules/type-declaration/src/Rector/Identical/FlipTypeControlToUseExclusiveTypeRector.php

```diff
@@ -125,7 +125,7 @@ private function processConvertToExclusiveType(array $types, Expr $expr, PhpDocI
         /** @var VarTagValueNode $tagValueNode */
         $tagValueNode = $phpDocInfo->getVarTagValueNode();
         $phpDocInfo->removeTagValueNodeFromNode($tagValueNode);
-        return new BooleanNot(new Instanceof_($expr, new FullyQualified($type)));
+        return new BooleanNot(new Instanceof_($expr, new Name($type)));
```

Comment:
This should work with FQN, as we use it everywhere in Rector by convention.

The name resolution of `Closure` must be fixed above, probably in `$type`.

Reply:
The type name returns actual value like in the doc, eg:

```php
@var stdClass|null
``` 

The types[0] will be 'stdClass'

and for :


```php
@var \stdClass|null
``` 

The types[0] will be '\stdClass'

any specific type converter for it?

Reply:
I think `StaticTypeMapper` would do here.


Proably `mapPHPStanPhpDocTypeToPHPStanType()` method

Reply:
implemented.